control-plane-operator/controllers/hostedcontrolplane/kubevirt_assets/files/controller.yaml

```diff
@@ -0,0 +1,125 @@
+kind: Deployment
+apiVersion: apps/v1
+metadata:
+  name: kubevirt-csi-controller
+  namespace: {{NAMESPACE}}
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: kubevirt-csi-driver
+  template:
+    metadata:
+      labels:
+        app: kubevirt-csi-driver
+    spec:
+      hostNetwork: true
+      serviceAccount: kubevirt-csi
+      priorityClassName: system-cluster-critical
+      nodeSelector:
+        node-role.kubernetes.io/master: ""
+      tolerations:
+        - key: CriticalAddonsOnly
+          operator: Exists
+        - key: node-role.kubernetes.io/master
+          operator: Exists
+          effect: "NoSchedule"
+      containers:
+        - name: csi-driver
+          imagePullPolicy: Always
+          image: quay.io/isaacdorfman/kubevirt-csi-driver:latest
+          args:
+            - "--endpoint=$(CSI_ENDPOINT)"
+            - "--namespace=kubevirt-csi-driver"
```

Comment:
this cli arg needs to be removed now if you want to use quay.io/dvossel/kubevirt-csi-driver:latest